cmake/3rdParty/Platform/Linux/BuiltInPackages_linux.cmake

```
@@ -12,54 +12,109 @@
+    ly_associate_package(PACKAGE_NAME ISPCTexComp-36b80aa-rev2-linux-aarch64                     TARGETS ISPCTexComp                 PACKAGE_HASH c29aafa32f13839a394424cf674b5cdb323fab22bcca43c38b43adfe13fc415c)
+    ly_associate_package(PACKAGE_NAME lz4-1.9.4-rev2-linux-aarch64                               TARGETS lz4                         PACKAGE_HASH 725ca4a02bcf961dc68fb525d0509c311536b5a0f0f9885244fab282cdc55d1f)
+    ly_associate_package(PACKAGE_NAME pyside2-5.15.2.1-py3.10-rev4-linux-aarch64                 TARGETS pyside2                     PACKAGE_HASH cc54c4783a645003a74e6a276c75b64b3eaee39f576423f4ebce0130621e2916)
+    ly_associate_package(PACKAGE_NAME SQLite-3.37.2-rev1-linux-aarch64                           TARGETS SQLite                      PACKAGE_HASH 5cc1fd9294af72514eba60509414e58f1a268996940be31d0ab6919383f05118)
+    ly_associate_package(PACKAGE_NAME AwsIotDeviceSdkCpp-1.15.2-rev1-linux-aarch64               TARGETS AwsIotDeviceSdkCpp          PACKAGE_HASH 0bac80fc09094c4fd89a845af57ebe4ef86ff8d46e92a448c6986f9880f9ee62)
+    ly_associate_package(PACKAGE_NAME vulkan-validationlayers-1.2.198-rev1-linux-aarch64         TARGETS vulkan-validationlayers     PACKAGE_HASH e67a15a95e14397ccdffd70d17f61079e5720fea22b0d21e135497312419a23f)
+
+    set(AZ_USE_PHYSX5 OFF CACHE BOOL "When ON PhysX Gem will use PhysX 5 SDK")
+    if(AZ_USE_PHYSX5)
+        ly_associate_package(PACKAGE_NAME PhysX-5.1.1-rev1-linux-aarch64                         TARGETS PhysX                       PACKAGE_HASH ba14e96fc4b85d5ef5dcf8c8ee58fd37ab92cfef790872d67c9a389aeca24c19)
+    else()
+        ly_associate_package(PACKAGE_NAME PhysX-4.1.2.29882248-rev5-linux-aarch64                TARGETS PhysX                       PACKAGE_HASH 7fa00d7d4f7532cf068246d4e424ce319529dfa09fb759d251676f2c59f6d50c)
+    endif()
+
+    # Certain packages are built against OpenSSL, so we must associate the proper ones based on their OpenSSL (Major) versions
+    if ("${OPENSSL_VERSION}" STREQUAL "")
+        message(FATAL_ERROR "OpenSSL not detected. The OpenSSL dev package is required for O3DE")
+    elseif ("${OPENSSL_VERSION}" VERSION_LESS "3.0.0")
+        ly_associate_package(PACKAGE_NAME AWSNativeSDK-1.9.50-rev2-linux-openssl-1-aarch64       TARGETS AWSNativeSDK                PACKAGE_HASH db83fb53946602495e586dfb1ef2dd52018d38d36e0f12c8baea3d1d87518e77)
+    else()
+        ly_associate_package(PACKAGE_NAME AWSNativeSDK-1.9.50-rev2-linux-openssl-3-aarch64       TARGETS AWSNativeSDK                PACKAGE_HASH 963d97090bf1a9347a4c8d512c393ff34961d3faba4eaa3a2d00e26b54ff36d4)
+    endif()
 else()
-	ly_associate_package(PACKAGE_NAME AWSNativeSDK-1.9.50-rev2-linux-openssl-3          TARGETS AWSNativeSDK                PACKAGE_HASH 9b9b5124791fb2f59b7362a95ed997944aff6cc928b9dede916e8968a09f23ff)
+    message(FATAL_ERROR "Unsupport linux architecture ${CMAKE_SYSTEM_PROCESSOR}")
```

typo: missing "ed " in "Unsupport"
```suggestion
    message(FATAL_ERROR "Unsupported linux architecture ${CMAKE_SYSTEM_PROCESSOR}")
```